pkgs/by-name/vo/voicevox-engine/package.nix

```diff
@@ -2,61 +2,57 @@
   lib,
   fetchFromGitHub,
   python3Packages,
-  replaceVars,
   voicevox-core,
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "voicevox-engine";
-  version = "0.23.0";
+  version = "0.24.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "VOICEVOX";
     repo = "voicevox_engine";
     tag = version;
-    hash = "sha256-kuWpLnDKRYcfV9FxYLeR6FmQFO2K12KxJx/Y/4MwhbM=";
+    hash = "sha256-LFbKnNv+NNfA6dvgVGr8fGr+3o5/sAyZ8XFZan2EJUY=";
   };
 
   patches = [
-    # the upstream package only uses poetry for dependency management, not for package definition
-    # this patch makes the package installable via poetry-core
-    (replaceVars ./make-installable.patch {
-      inherit version;
-    })
+    # this patch makes the package installable via hatchling
```

Comment:
Please consider opening an upstream PR for this

Reply:
Upstream uses `pyinstaller` which is incompatible with nixpkgs.
I'm not sure they would want to have two different ways of installing stuff.

Reply:
Ok, thanks